### PR TITLE
Fixed 0-length read in text parser

### PR DESCRIPTION
**Pull request by Onager**
Depends on dfvfs 20180831

### CODE REVIEW

**Comment by joachimmetz** [comment]
The tests are currently failing

**Comment by Onager** [comment]
Tests are failing until https://github.com/log2timeline/dfvfs/pull/311 is committed and released.

**Comment by joachimmetz** [comment]
> Tests are failing until log2timeline/dfvfs#311 is committed and released.

This would also require a new dfvfs release

**Comment by Onager** [comment]
@aguilajesus can you confirm that the maximum length for the Santa parser is ok?

**Comment by aguilajesus** [comment]
@Onager yes, it should cover the majority of cases. the only use cases I have seen passing that number is active developers and compilers.

**Comment by joachimmetz** [comment]
Pushed new dfvfs to l2tbinaries and GIFT PPA, and restarted tests.

File: plaso/dependencies.py [hunk 1]
```diff
@@ -145,12 +145,6 @@ def _CheckPythonModule(
       map(int, _VERSION_SPLIT_REGEX.split(minimum_version)))
 
   if module_version_map < minimum_version_map:
-    if not is_required:
```
Comment (joachimmetz) [inline plaso/dependencies.py]: You'll need to re-add these checks since they've not been integrated in l2tdevtools yet

**Comment by joachimmetz** [comment]
Addressed some regression, due to auto-generation, waiting for the tests to complete.

**Pull request merged**